include/odp/api/spec/packet_io.h

```diff
@@ -178,7 +178,8 @@ typedef struct odp_pktin_vector_config_t {
 	 *
 	 * When true, packet input vector is enabled and configured with vector
 	 * config parameters. Otherwise, packet input vector configuration
-	 * parameters are ignored.
+	 * parameters are ignored. When vectors are enabled, all packets
+	 * will be delivered as vectors. Default value is false.
```

Comment:
Agreed to document that even when vectors are enabled packets may be also received